lib/install/validate/validator.go

```diff
@@ -280,19 +289,40 @@ func (v *Validator) target(ctx context.Context, input *data.Data, conf *config.V
 		conf.UserPassword = targetURL.User.String()
 	}
 
-	// bridge network params
-	var err error
-	_, conf.BridgeIPRange, err = net.ParseCIDR(input.BridgeIPRange)
-	if err != nil {
-		v.NoteIssue(fmt.Errorf("Error parsing bridge network ip range: %s. Range must be in CIDR format, e.g., 172.16.0.0/12", err))
-	}
+	// check if host is managed by VC
+	v.managedbyVC(ctx)
 
 	conf.Target = *targetURL
 	conf.Insecure = input.Insecure
 
 	// TODO: more checks needed here if specifying service account for VCH
 }
 
+func (v *Validator) managedbyVC(ctx context.Context) {
+	defer trace.End(trace.Begin(""))
+
+	if v.IsVC() {
+		return
+	}
+	host, err := v.Session.Finder.DefaultHostSystem(ctx)
+	if err != nil {
+		v.NoteIssue(fmt.Errorf("Failed to get host system: %s", err))
+		return
+	}
+
+	var mh mo.HostSystem
+
+	if err = host.Properties(ctx, host.Reference(), []string{"summary.managementServerIp"}, &mh); err != nil {
+		v.NoteIssue(fmt.Errorf("Failed to get host properties: %s", err))
+		return
+	}
+
+	if ip := mh.Summary.ManagementServerIp; ip != "" {
+		v.NoteIssue(fmt.Errorf("Target is managed by vCenter server %q, please change --target to vCenter or standalone ESXi address", ip))
```

Comment:
maybe `Target is managed by vCenter server %q, please change --target to the vCenter address or select a standalone ESXi`